modules/SerialCommHub/tiny_modbus_rtu.cpp

```diff
@@ -64,117 +64,16 @@ static void append_checksum(uint8_t* msg, int msg_len) {
 static bool validate_checksum(const uint8_t* msg, int msg_len) {
     if (msg_len < 5)
         return false;
+
+    EVLOG_info << hexdump(msg, msg_len);
```

Comment:
Looks like this debug needs to be removed before merge?

Reply:
Please make sure you comment only on the latest version, not on intermediary commits! 
This has already been fixed by the next commit.